include/zephyr/drivers/input.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_INPUT_H_
+#define ZEPHYR_INCLUDE_DRIVERS_INPUT_H_
+
+/**
+ * @brief Input Driver Interface
+ * @defgroup input_driver_interface Input Driver Interface
+ * @ingroup io_interfaces
+ * @{
+ */
+
+#include <zephyr/kernel.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Input device driver API. */
+__subsystem struct input_driver_api {
+};
```

Comment:
note: if we don't need an API right now, let's just remove this.

Reply:
You sure about it? It'd be convenient if we end up adding something. Otherwise I guess we can drop the whole header.

Reply:
You know what. Dropped the whole file, we can always add it later.